### PR TITLE
TS: use BOARD_NAME in Test and Version commands reply

### PR DESCRIPTION
**Pull request by rusefillc**

### CODE REVIEW

File: firmware/console/binary/tunerstudio.cpp [hunk 1]
```diff
@@ -225,10 +223,10 @@ static void handleTestCommand(TsChannelBase* tsChannel) {
 	 * extension of the protocol to simplify troubleshooting
 	 */
 	tunerStudioDebug(tsChannel, "got T (Test)");
-	chsnprintf(testOutputBuffer, sizeof(testOutputBuffer),  VCS_VERSION "\r\n");
+	chsnprintf(testOutputBuffer, sizeof(testOutputBuffer), "%s\r\n", BOARD_NAME);
```
Comment (mck1117) [inline firmware/console/binary/tunerstudio.cpp:226]: fun fact, c/c++ will concatenate adjacent strings

```c++
const char* foo1 = "HELLO" " " " WORLD";
const char* foo2 = "HELLO WORLD";
```

these string constants are identical

Comment (mck1117) [reply firmware/console/binary/tunerstudio.cpp:226]: (so we don't actually need a formatted printf at all, just concat them"

Comment (mck1117) [reply firmware/console/binary/tunerstudio.cpp:226]: @dron0gus @rusefillc ping

Comment (rusefillc) [reply firmware/console/binary/tunerstudio.cpp]: source code is now two bytes smaller and less uniform, please review

Comment (rusefillc) [reply firmware/console/binary/tunerstudio.cpp]: source code is now six bytes smaller and uniform, please review

**Pull request merged**